### ISSUE TITLE
Add an optional checksum field to occurrences

### ISSUE THREAD

**Issue by R2wenD2** [grafeas/grafeas#26]

**Comment by vbatts** [comment]
left up to input sources as to what the checksum pertains to?

**Comment by R2wenD2** [comment]
This is open here for more discussion. This could potentially be a structured checksum field.
As an example, in the case of a docker container image, a user might not care about the entire url (which registry/project the image came from), but only about the digest. So the checksum is meant to be an alternative identifier to resourceUrl.

**Comment by hpejustin** [comment]
@R2wenD2 yes, in some case, user use latest tag, but the content may be not the same, so they need checksum to identify them.

**Comment by R2wenD2** [comment]
Vincent, can you update the proto to include the new fields?
